.github/workflows/deploy-prod.yml

```
@@ -0,0 +1,128 @@
+      ssh_port: ${{ secrets.PROD_SSH_PORT }}
+      ssh_host_fingerprint: ${{ secrets.PROD_SSH_HOST_FINGERPRINT }}
+      module: ${{ github.event.inputs.module }}
+      playbook: playbooks/deploy.yml
+      inventory_path: inventories/prod/hosts.yml
+      success_title: Deploy Prod Succeeded
+      failure_title: Deploy Prod Failed
+      notify_description: |
+        Module: ${{ github.event.inputs.module }}
+        Version: ${{ github.event.inputs.version }}
+        Actor: ${{ github.actor }}
+      image: ${{ secrets.PROD_DOCKER_LOGIN_USERNAME }}/beat-${{ github.event.inputs.module }}:${{ github.event.inputs.version }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)